tools/BuildAssets/psModules/CodeGenerationModules/AutoRestCodeGenerationModule/AutoRestCodeGenerationModule.psm1

```diff
@@ -338,13 +338,13 @@ function Start-AutoRestCodeGeneration {
     )
 
     if(-not [string]::IsNullOrWhiteSpace($SdkDirectory)) {
-        Start-CodeGeneration -ResourceProvider $ResourceProvider -SdkDirectory $SdkDirectory -Namespace $Namespace -ConfigFileTag $ConfigFileTag -SpecsRepoFork $SpecsRepoFork -SpecsRepoName $SpecsRepoName -SpecsRepoBranch $SpecsRepoBranch -SdkGenerationType $SdkGenerationType
+        Start-CodeGeneration -ResourceProvider $ResourceProvider -SdkDirectory $SdkDirectory -Namespace $Namespace -ConfigFileTag $ConfigFileTag -SpecsRepoFork $SpecsRepoFork -SpecsRepoName $SpecsRepoName -SpecsRepoBranch $SpecsRepoBranch -SdkGenerationType $SdkGenerationType -AutoRestVersion $AutoRestVersion
```

Comment:
@dsgouda currently there are total 11 parameters exposed for Start-AutoRestCodeGeneration

1. when you pass these parameters to Start-CodeGeneration, can you pass it in the same order you have exposed your parameters?

Currently you have 11 parameters exposed in Start-AutoRestCodeGeneration but you are passing 10 to Start-CodeGeneration. Having them passed in the same order helps in making sure things are getting passed.

Reply:
I thought named parameters worked irrespective of the order, am I missing something?